crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -425,7 +437,7 @@ async fn c8y_mapper_syncs_pending_alarms_on_startup() {
     //     .unwrap();
 
     // Restart the C8Y Mapper
-    let _ = start_c8y_mapper(broker.port).await.unwrap();
+    let c8y_mapper_re = start_c8y_mapper(broker.port).await.unwrap();
```

Comment:
What is the meaning of `_re` suffix?

I think that rebinding the name `c8y_mapper` would be clearer.

Reply:
I mean the restarted mapper, thought of having a different handle.